lua/entities/gmod_wire_expression2/core/vector.lua

```diff
@@ -624,11 +628,11 @@ end
 __e2setcost( 15 )
 
 e2function string pointContents( vector point )
-	return cache_concatenated_parts[util.PointContents( Vector(point[1],point[2],point[3]))]
+	return cache_concatenated_parts[util.PointContents( Vector(point[1],point[2],point[3]))] or ""
```

Comment:
This change is useless because the cache has a metatable which ensures that it will never return anything other than a string. Same is true for the function below this one.